src/StripeController.php

```diff
@@ -117,7 +117,10 @@ public function execute(\Payment $payment) {
     $api = Api::init($payment->method);
     $intent = $this->fetchIntent($payment, $api);
     entity_save('payment', $payment);
-    return $this->createSubscriptions($payment, $api, $intent);
+
+    // The `payment_method` is expanded for sepa but not credit card payments.
+    $stripe_pm = $intent->payment_method->id ?? $intent->payment_method;
+    return $this->createSubscriptions($payment, $api, $stripe_pm);
```

Comment:
The same value is called `$payment_method_id` in `createCustomer()` above. Using the same variable name in both cases would make it more readible.